Size prop added to ImageInput component (#1260)
sumup-oss/circuit-ui: .changeset/nice-toys-reply.md

---
'@sumup/circuit-ui': patch
---

> `buttonSize` prop added to ImageInput component and giga Avatar border radius was changed to byte size

Reviewer: Nitpicks for the changelog:

```suggestion
Adjusted the border radius of the `Avatar` component when `size="giga"` and `variant="object"`.
```

And let's make a second changeset for the other change (it's a minor), you can run `yarn changeset` again and I'd suggest something like:

```md
---
'@sumup/circuit-ui': minor
---

Added an optional `buttonSize` prop to the `ImageInput` component to change the action button's size.
```
